From cbf0ba59e05cffee9bdc3fa643ed593e3311fbdc Mon Sep 17 00:00:00 2001
From: Calvin Johnson <calvin.johnson@nxp.com>
Date: Thu, 9 May 2019 17:11:08 +0530
Subject: [PATCH 35/40] configs: ls1043avnprdb: Update EEPROM config

On VNPRDB, eeprom is located at i2c addr 0x50.
Enable 16-bit EEPROM addressing for accessing M24512 EEPROM.

Signed-off-by: Calvin Johnson <calvin.johnson@nxp.com>
---
 include/configs/ls1043avnprdb.h | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/include/configs/ls1043avnprdb.h b/include/configs/ls1043avnprdb.h
index 60031f4896..2b6d69cafc 100755
--- a/include/configs/ls1043avnprdb.h
+++ b/include/configs/ls1043avnprdb.h
@@ -224,10 +224,8 @@ board/freescale/ls1043ardb/ls1043avnprdb_rcw_emmc_nand.cfg
 #define CONFIG_ID_EEPROM
 #define CONFIG_SYS_I2C_EEPROM_NXID
 #define CONFIG_SYS_EEPROM_BUS_NUM		0
-#define CONFIG_SYS_I2C_EEPROM_ADDR		0x53
-#define CONFIG_SYS_I2C_EEPROM_ADDR_LEN		1
-#define CONFIG_SYS_EEPROM_PAGE_WRITE_BITS	3
-#define CONFIG_SYS_EEPROM_PAGE_WRITE_DELAY_MS	5
+#define CONFIG_SYS_I2C_EEPROM_ADDR		0x50
+#define CONFIG_SYS_I2C_EEPROM_ADDR_LEN		2
 #endif
 
 
-- 
2.17.1

